website/src/app/kb/quickstart/readme.mdx

```diff
@@ -0,0 +1,3 @@
+# Quickstart Guide
+
+{/* TODO */}
```

Comment:
@bmanifold Can probably just copy-paste the Quickstart guide draft here.

Reply:
Just pushed a commit to this branch with the Quickstart guide content.  If you'd rather it be in a separate PR, feel free to reset and force push over it.

Reply:
👍